picgo@2.3.0: download via aria2 failed

```
Updating 'picgo' (2.2.2 -> 2.3.0)
Downloading new version
Starting download with aria2 ...
Download: 09/12 09:45:58 [ERROR] CUID#8 - Download aborted. URI=https://github.com/Molunerfinn/PicGo/releasesDownload: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/Molunerfinn/PicGo/releases/dDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 09a9a9|ERR |       0B/s|D:/Applications/Scoop/cache/picgo#2.3.0#https_github.com_Molunerfinn_PicGo_releases_download_v2.3.0_picgo-setup-2.3.0.exe_dl.7z
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/Molunerfinn/PicGo/releases/download/v2.3.0/picgo-setup-2.3.0.exe#/dl.7z
    referer=https://github.com/Molunerfinn/PicGo/releases/download/v2.3.0/picgo-setup-2.3.0.exe#/
    dir=D:\Applications\Scoop\cache
    out=picgo#2.3.0#https_github.com_Molunerfinn_PicGo_releases_download_v2.3.0_picgo-setup-2.3.0.exe_dl.7z

ERROR & 'D:\Applications\Scoop\apps\aria2\current\aria2c.exe' --input-file='D:\Applications\Scoop\cache\picgo.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=16 --max-connection-per-server=16 --min-split-size=1M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=35636 --continue --summary-interval 0 --all-proxy='127.0.0.1:1081' --check-certificate=false
```


It seems the filename in the download page has changed.  
It should be `PicGo-Setup-2.3.0-x64.exe` instead of `picgo-setup-2.3.0.exe`

![image](https://user-images.githubusercontent.com/12221881/132967990-ddb776e5-5cfc-4b8b-ae91-ab6c9354ecdb.png)

--- a/bucket/picgo.json
+++ b/bucket/picgo.json
@@ -13,7 +13,7 @@
         },
         "32bit": {
             "url": "https://github.com/Molunerfinn/PicGo/releases/download/v2.3.0/picgo-setup-2.3.0-ia32.exe#/dl.7z",
-            "hash": "78cbd6ea2618777f369304120cd8d7341f960017a89e5e017f9430a7a3c6d41c"
+            "hash": "sha512:c81ec09a612de90c3982dc508ed81620702c577a3288fa15adf71dea0adee7fe36f30f92ed4a1a5a64d5c971fac68f603bd1cd4e551c3adf742360c4da16bc0e"
         }
     },
     "installer": {
@@ -36,16 +36,16 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/Molunerfinn/PicGo/releases/download/v$version/picgo-setup-$version-x64.exe#/dl.7z",
-                "hash": {
-                    "url": "https://github.com/Molunerfinn/PicGo/releases/download/v$version/latest.yml",
-                    "mode": "extract",
-                    "regex": "(?sm)exe.*sha512: $base64"
-                }
+                "url": "https://github.com/Molunerfinn/PicGo/releases/download/v$version/picgo-setup-$version-x64.exe#/dl.7z"
             },
             "32bit": {
                 "url": "https://github.com/Molunerfinn/PicGo/releases/download/v$version/picgo-setup-$version-ia32.exe#/dl.7z"
             }
+        },
+        "hash": {
+            "url": "https://github.com/Molunerfinn/PicGo/releases/download/v$version/latest.yml",
+            "mode": "extract",
+            "regex": "(?sm)exe.*sha512: $base64"
         }
     }
 }

--- a/bucket/picgo.json
+++ b/bucket/picgo.json
@@ -6,8 +6,16 @@
         "identifier": "996ICU",
         "url": "https://github.com/Molunerfinn/PicGo/blob/dev/LICENSE"
     },
-    "url": "https://github.com/Molunerfinn/PicGo/releases/download/v2.3.0/picgo-setup-2.3.0.exe#/dl.7z",
-    "hash": "sha512:c81ec09a612de90c3982dc508ed81620702c577a3288fa15adf71dea0adee7fe36f30f92ed4a1a5a64d5c971fac68f603bd1cd4e551c3adf742360c4da16bc0e",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Molunerfinn/PicGo/releases/download/v2.3.0/picgo-setup-2.3.0-x64.exe#/dl.7z",
+            "hash": "sha512:c81ec09a612de90c3982dc508ed81620702c577a3288fa15adf71dea0adee7fe36f30f92ed4a1a5a64d5c971fac68f603bd1cd4e551c3adf742360c4da16bc0e"
+        },
+        "32bit": {
+            "url": "https://github.com/Molunerfinn/PicGo/releases/download/v2.3.0/picgo-setup-2.3.0-ia32.exe#/dl.7z",
+            "hash": "78cbd6ea2618777f369304120cd8d7341f960017a89e5e017f9430a7a3c6d41c"
+        }
+    },
     "installer": {
         "script": [
             "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
@@ -26,11 +34,18 @@
         "regex": "releases/tag/v([\\d.]+)\""
     },
     "autoupdate": {
-        "url": "https://github.com/Molunerfinn/PicGo/releases/download/v$version/picgo-setup-$version.exe#/dl.7z",
-        "hash": {
-            "url": "https://github.com/Molunerfinn/PicGo/releases/download/v$version/latest.yml",
-            "mode": "extract",
-            "regex": "(?sm)exe.*sha512: $base64"
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/Molunerfinn/PicGo/releases/download/v$version/picgo-setup-$version-x64.exe#/dl.7z",
+                "hash": {
+                    "url": "https://github.com/Molunerfinn/PicGo/releases/download/v$version/latest.yml",
+                    "mode": "extract",
+                    "regex": "(?sm)exe.*sha512: $base64"
+                }
+            },
+            "32bit": {
+                "url": "https://github.com/Molunerfinn/PicGo/releases/download/v$version/picgo-setup-$version-ia32.exe#/dl.7z"
+            }
         }
     }
 }

